### PR TITLE
Launchpad: fullscreen method refactor

### PR DESCRIPTION
**Pull request by andres-blanco**
Related to https://github.com/Automattic/wp-calypso/issues/80140

## Proposed changes:

Rename `wpcom_get_launchpad_is_enabled` to `wpcom_launchpad_get_fullscreen_enabled`.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Verify tests pass correctly
* Verify there's no other usages where the function needs to be replaced. I opengroked in all projects, but a double check wont hurt

### CODE REVIEW

File: projects/packages/jetpack-mu-wpcom/changelog/launchpad-fullscreen-method-refactor [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: simple refactor
```
Comment (daledupreez) [inline projects/packages/jetpack-mu-wpcom/changelog/launchpad-fullscreen-method-refactor]: I think it's worth being explicit about the refactor.
```suggestion
Comment: Rename 'wpcom_get_launchpad_is_enabled()' to 'wpcom_launchpad_get_fullscreen_enabled()'
```

**Review by daledupreez — DISMISSED** [review]
Thanks for tackling this, and breaking the work down into smaller PRs. 💖 

The changes look good to me, but I think we should improve the changelog entry (which I didn't review for the other PRs... 😳 ). I also couldn't find any other references to this function.

File: projects/packages/jetpack-mu-wpcom/changelog/launchpad-fullscreen-method-refactor (hunk 1 above)
Comment (andres-blanco) [reply projects/packages/jetpack-mu-wpcom/changelog/launchpad-fullscreen-method-refactor]: Completely agree 😁

**Review by daledupreez — APPROVED** [review]
Updates look good to me!

**Pull request merged**